frontend/src/core/user/components/UserMenu.test.js

```diff
@@ -58,6 +61,75 @@ describe('<UserMenuBase>', () => {
         expect(wrapper.find(SignOut)).toHaveLength(0);
     });
 
+    it('shows upload & download menu items', () => {
+        const wrapper = shallow(
+            <UserMenuBase
+                isTranslator={ true }
+                user={ { isAuthenticated: true } }
+                parameters={ { locale: 'mylocale', project: 'myproject', resource: 'myresource' } }
+            />
+        );
```

Comment:
I'd recommend you refactor this into a function at the start of the file, and replace all instances of the same line in your tests. 

Here's an example of such a function, with default values for parameters, and to which you can pass only some of the params: https://github.com/mozilla/pontoon/blob/master/frontend/src/modules/editor/components/Editor.test.js#L23